src/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,75 @@
+import {
+	AfterContentInit,
+	Component,
+	ContentChildren,
+	Input,
+	QueryList,
+	ViewChild,
+	ViewContainerRef
+} from "@angular/core";
+
+import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
+
+@Component({
+	selector: "ibm-breadcrumb",
+	template: `
+	<nav #nav class="bx--breadcrumb"
+		[ngClass]="{
+			'bx--breadcrumb--no-trailing-slash' : noTrailingSlash
+		}"
+		[attr.aria-label]="ariaLabel">
+		<ng-content *ngIf="noOverFlow(); else overflow"></ng-content>
+		<ng-template #overflow>
+			<ibm-breadcrumb-item [href]="first?.href" text="{{first?.text}}">
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item>
+				<ibm-overflow-menu>
+					<ibm-overflow-menu-option *ngFor="let item of overflowItems">
+						<ibm-breadcrumb-item [href]="item?.href" [text]="item?.text">
+						</ibm-breadcrumb-item>
+					</ibm-overflow-menu-option>
+				</ibm-overflow-menu>
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item [href]="secondLast?.href" [text]="secondLast?.text">
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item [href]="last?.href" [text]="last?.text">
+			</ibm-breadcrumb-item>
+		</ng-template>
+	</nav>`
+})
+export class BreadcrumbComponent implements AfterContentInit {
+	@Input() noTrailingSlash = false;
+
+	@Input() ariaLabel: string;
+
+	@Input() overflowThreshold = 4;
+
+	@ContentChildren(BreadcrumbItemComponent) items !: QueryList<BreadcrumbItemComponent>;
+
+	first: BreadcrumbItemComponent = null;
```

Comment:
These 4 should really be getters returning parts of `items` array. There are more reasons for that, the most prominent being: what happens when `items` change on the fly?

Getters could use (conditionally) indexes, or `slice`s